templates/taxi/car_list.html

```diff
@@ -3,15 +3,28 @@
 {% block content %}
   <h1>Car list</h1>
   {% if car_list %}
-    <ul>
-      {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
-      {% endfor %}
-    </ul>
+    <table class="table">
+    <tr>
+      <th>ID</th>
+      <th>Model</th>
+      <th>Manufacturer</th>
+    </tr>
+    {% for car in car_list %}
+    <tr>
+      <td>
+        <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
```

Comment:
There is an extra space before the closing quotation mark in the `href` attribute of the car detail link. This might cause issues with URL resolution.